docs/api/ipc-renderer.md

```diff
@@ -32,6 +32,14 @@ The `ipcRenderer` module has the following method to listen for events and send
 Listens to `channel`, when a new message arrives `listener` would be called with
 `listener(event, args...)`.
 
+### `ipcRenderer.off(channel, listener)`
+
+* `channel` string
+* `listener` Function
+  * `...args` any[]
```

Comment:
```suggestion
  * `event` [IpcRendererEvent][ipc-renderer-event]
  * `...args` any[]
```

Doesn't really make a functional difference due to the argument spread, but I think we should be consistent and accurate here. Same change needed for `ipcRenderer.removeListener`.